src/strategies/ocean-dao-brightid/index.ts

```diff
@@ -15,6 +15,9 @@ export async function strategy(
   options,
   snapshot
 ) {
+  provider = getProvider(options.chainId);
+  snapshot = options.chainBlock;
```

Comment:
Any reason to use a snapshot defined in the strategy param? This will not make it easy to use, it will force to update space settings with the proper snapshot block number everytime you want to make a new proposal.

Reply:
Hey there! I'm planning to pass strategy params when creating the proposal using the API. I don't think it will be a problem.

Reply:
@trizin Problem is that users have to change the strategies settings everytime before a new proposal

Reply:
Thanks, I removed this part.